lib/clearance/password_strategies/sha1.rb

```diff
@@ -3,13 +3,21 @@ module PasswordStrategies
     module SHA1
       require 'digest/sha1'
 
+      DEPRECATION_MESSAGE = "[DEPRECATION] The SHA1 password strategy " \
+        "has been deprecated and will be removed from Clearance 2.0. BCrypt " \
+        "is the only officially supported strategy, though you are free to " \
+        "provide your own. To continue using this strategy add " \
+        "clearance-deprecated_password_strategies to your Gemfile."
```

Comment:
Would you recommend bucking the convention here?

On Mon, May 11, 2015 at 5:12 PM, Joe Ferris <notifications@github.com>
wrote:

> In lib/clearance/password_strategies/sha1.rb
> <https://github.com/thoughtbot/clearance/pull/558#discussion_r30082672>:
>
> > @@ -3,13 +3,21 @@ module PasswordStrategies
> >      module SHA1
> >        require 'digest/sha1'
> >
> > +      DEPRECATION_MESSAGE = "[DEPRECATION] The SHA1 password strategy " \
> > +        "has been deprecated and will be removed from Clearance 2.0. BCrypt " \
> > +        "is the only officially supported strategy, though you are free to " \
> > +        "provide your own. To continue using this strategy add " \
> > +        "clearance-deprecated_password_strategies to your Gemfile."
>
> FYI, it was originally factory_girl-rails but enough people complained
> that I renamed it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/thoughtbot/clearance/pull/558/files#r30082672>.
>